bot/utils/decorators.py

```diff
@@ -203,20 +203,36 @@ def predicate(ctx: Context) -> bool:
 
         kwargs = default_kwargs.copy()
 
-        # Update kwargs based on override
-        if hasattr(ctx.command.callback, "override"):
+        # determine which command's overrides we will use
+        # as we have groups, we want to ensure that group commands inherit from the parent
+        overridden_command: Union[commands.Command, commands.Group] = None
+        for command in [ctx.command, *ctx.command.parents]:
+            print(command)
```

Comment:
Is this intentional? If not,
```suggestion
```

Reply:
https://github.com/python-discord/sir-lancebot/pull/981/commits/1e69c3a4c7d799c5a1450e59c9d390097778d3b8